inc/functions-cookies.php

```diff
@@ -0,0 +1,75 @@
+<?php
+// These functions are going to be moved into the plugin
+// upon cross site roll-out
+
+function delete_GA_cookies() {
+	$domain = 'nationalarchives.gov.uk';
+	$cookie_list = ['_ga', '_gid', '_gat_UA-2827241-22', '_gat_UA-2827241-1'];
+    
+	if (isset($_SERVER['HTTP_COOKIE'])) {
+		$cookies = explode(';', $_SERVER['HTTP_COOKIE']);
+		foreach($cookies as $cookie) {
+			$parts = explode('=', $cookie);
+			$name = trim($parts[0]);
+			foreach($cookie_list as $single_cookie) {
+				if($name == $single_cookie) {
```

Comment:
I think the indentation is a bit off here.

Reply:
I actually wonder why it looks like that because it doesn't reflect my actual code format.
![Screenshot 2021-03-25 at 12 07 57](https://user-images.githubusercontent.com/5245264/112470731-e8624d80-8d62-11eb-88fb-bcda3c745941.png)